build-support/bin/release.sh

```diff
@@ -672,21 +674,62 @@ function activate_twine() {
 
 function build_pex() {
   # Builds a pex from the current UNSTABLE version.
+  # If $1 == "build", builds a pex just for this platform, from source.
+  # If $1 == "fetch", fetches the linux and OSX wheels which were built on travis.
+  local mode="$1"
 
   local linux_platform="linux_x86_64"
+  local osx_platform="macosx_10.10_x86_64"
+
+  case "${mode}" in
```

Comment:
This mode/switch approach feels like it results in more local variables to keep in your head than explicit function calls would.

Reply:
By explicit function call, do you mean effectively copy-pasting `build_pex` into a `build_local_pex` function? That's what I originally did, but the parameters needed for code-sharing felt pretty small (just the `platforms` array and `dest` string)... But maybe I'm missing the thing you're proposing?

Reply:
> By explicit function call, do you mean effectively copy-pasting build_pex into a build_local_pex function?

I meant more like decomposing `build_pex` into more functions: maybe "functions to fetch/build whls" and then "function to actually build the pex". But ignore me: feel free to land as is. More important is just that we eventually invest to port this.